org.flathub.p3x-onenote.yml

```diff
@@ -0,0 +1,77 @@
+app-id: org.flathub.p3x-onenote
```

Comment:
You cannot use `org.flathub` as ID, you don't own the domain flathub.org. Please read the app id guidelines in the checklist

Reply:
This is blocking the submission btw